### ISSUE TITLE
Error with Android Icon

### ISSUE THREAD

**Issue by fletchy95** [mlach88/StockTrends#10]
For some reason I am getting some error that is saying:
    "Error APT0000: No resource found that matches the given name (at 'icon' with value 
    '@drawable/icon'). (APT0000) (StockTrends20.Droid)"

I have done research, and I attempted to change file names but now it says the file already exists.  I'm still looking in to this issue.  Help would be appreciated if someone sees something wrong.  I will also talk to professor.  View screenshot below.

<img width="1440" alt="screen shot 2018-04-25 at 8 21 18 pm" src="https://user-images.githubusercontent.com/32146593/39279309-3b8d6704-48c6-11e8-8491-c4b4123dcdec.png">

**Comment by fletchy95** [comment]
Recent push to JackV2 may have fixed error.  I found that even though files were deleted from the Xamarin Solution, they were not deleted from the File tree.  I had to manually go in to the file tree and drag files to the trash.  Once doing this I renamed the file from what it was before to 'icon.png'.  There appears to be no error.  But I will have @mlach88 run the solution on the android side to confirm.
